blackjack.py

```diff
@@ -242,11 +269,30 @@ def play_again():
             continue
 
 
-def retotal(h):
-    new_total = 0
-    for card in h:
-        new_total += card_point(card)
-    return new_total
+def hand_total(hand):
+    total = 0
+
+    # Evaluate card points and add them to total.
+    for card in hand:
+        r = card.rank
+        points = card_point(card)
+
+        # If an Ace (11) would cause a bust, change to 1. If not Ace remains 11.
+        if r == Rank.Ace:
+            if total + points > 21:
+                total -= 10
+        total += points
+
+    # If you have more than 2 cards in hand and total > 21, change each Ace to 1.
+    if len(hand) > 2 and total > 21:
```

Comment:
Why do you check if the `len(hand) > 2` ? Is that a necessary check?

Reply:
If i remover len(hand) then a hand with [A, A] will return a total of 2.

Reply:
resolved